subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/RealLifeAndroidBuildPerformanceTest.groovy

```diff
@@ -19,18 +19,34 @@ package org.gradle.performance.regression.android
 import org.gradle.performance.categories.PerformanceExperiment
 import org.gradle.performance.fixture.BuildExperimentInvocationInfo
 import org.gradle.performance.fixture.BuildExperimentListenerAdapter
+import org.gradle.performance.fixture.CrossVersionPerformanceTestRunner
+import org.gradle.performance.mutator.ApplyAbiChangeToJavaSourceFileMutator
+import org.gradle.performance.mutator.ApplyNonAbiChangeToJavaSourceFileMutator
 import org.gradle.util.GFileUtils
 import org.junit.experimental.categories.Category
 import spock.lang.Unroll
 
 class RealLifeAndroidBuildPerformanceTest extends AbstractAndroidPerformanceTest {
+    private static final SANTA_TRACKER = new AndroidTestProject(
+        templateName: 'santaTrackerAndroidBuild',
+        memory: '1g',
+        taskForChange: ':santa-tracker:assembleDebug',
+        fileToChange: 'snowballrun/src/main/java/com/google/android/apps/santatracker/doodles/snowballrun/BackgroundActor.java'
```

Comment:
I like the extraction of the `template` and `memory` values.

I'm not so sure about `taskForChange` and `fileToChange`, since they are just for this one kind of test. Maybe extract them to constants?

Another option would be to introduce something like `AndroidIncrementalTestProject` or something with these fields. It could configure the incremental execution in `configureIncremental()`, too.

Reply:
I added two constants for now.